TWTValidation/Validators/Value Validators/TWTStringValidator.m

```diff
@@ -257,6 +263,59 @@ - (BOOL)validateValue:(id)value error:(out NSError *__autoreleasing *)outError
 
 #pragma mark
 
+@implementation TWTBoundedComposedCharacterLengthStringValidator
+
+- (BOOL)validateValue:(id)value error:(out NSError *__autoreleasing *)outError
+{
+    TWTValueValidator *stringValidator = [TWTValueValidator valueValidatorWithClass:[NSString class] allowsNil:self.allowsNil allowsNull:self.allowsNull];
```

Comment:
No need to do this. You are a string validator. I think you can just invoke super, right?